src/main/java/uk/gov/justice/digital/delius/data/api/Attendance.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.annotations.ApiModelProperty;
+import java.time.LocalDate;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class Attendance {
+
+    @ApiModelProperty(required = true)
+    private final Long contactId;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
```

Comment:
What does this annotation do? Would the general Jackson config do the same thing or does it affect Swagger documentation ?

Reply:
Wasn't sure whether I wanted to go to do anything general

Reply:
Removed. Still got date as required.